react-frontend/src/Authentication/Registration.js

```diff
@@ -9,21 +9,35 @@ const Registration = () => {
   const [user, setUser] = useState({
     username: '',
     password: '',
+    confirm_password: '',
   });
   const [register, setRegister] = useState(false);
 
   const handleChange = (event) => {
     const { name, value } = event.target;
-    if (name === 'password') { setUser({ username: user.username, password: value }); } else setUser({ username: value, password: user.password });
+    console.log(name, value);
+    if (name === 'password') {
+      setUser({ username: user.username, password: value });
+    } else if (name === 'confirm_password') {
+      setUser({ username: user.username, password: user.password, confirm_password: value });
+    } else {
+      setUser(
+        { username: value, password: user.password, confirm_password: user.confirm_password },
+      );
+    }
   };
 
   const submitForm = async (event) => {
     event.preventDefault();
-    try {
-      await axios.post('http://localhost:8000/register', user);
+    if (user.password !== user.confirm_password) {
+      setUser({ username: '', password: '', confirm_password: '' });
```

Comment:
We may want to redirect the user to the home page after they have created their user